src/pages/Accounts/Details/AccountDetailsContainerScreen.tsx

```diff
@@ -333,10 +333,9 @@ const AccountDetailsContainerScreen: React.FC<Props> = ( { navigation } ) => {
   )
 }
 
-const styles = StyleSheet.create( {
+const styles = StyleSheet.create({
   rootContainer: {
     paddingTop: 20,
-    flex: 1,
```

Comment:
@CypherPoet removing this flex:1 made the initial animated refresh work again. Please review and let me know if removing this can have any side effects.